.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,111 @@
+# This is a reusable workflow for running ShellCheck,
+# a linter for shell scripts (https://shellcheck.net).
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on: 
+  workflow_call:
+    inputs:
+
+      root-dirs:
```

Comment:
Should it default to just searching the whole repository?

Reply:
Agreed. I flipped from this being an include-list to an exclude-list.